src/tigerbeetle/main.zig

```diff
@@ -66,37 +66,72 @@ pub fn main() !void {
 
     const allocator = arena.allocator();
 
-    var time_os: TimeOS = .{};
-    const time = time_os.time();
-
     var arg_iterator = try std.process.argsWithAllocator(allocator);
     defer arg_iterator.deinit();
 
     var command = cli.parse_args(&arg_iterator);
 
     switch (command) {
-        .inspect, .version => {},
+        .inspect => {},
+        .version => |*args| {
+            try Command.version(allocator, args.verbose);
+            return; // Exit early before initializing IO.
+        },
         inline else => |*args| {
             if (args.log_debug) {
                 log_level_runtime = .debug;
             }
         },
     }
```

Comment:
I'd maybe write this as

```zig
    if (command == .versoin) {
        try Command.version(allocator, command.version.verbose);
        return; // Exit early before initializing IO.
    }

    log_level_runtime = switch (command) {
        .version => unreachable,
        .inspect => .info,
        inline else => |*args| if (args.log_debug) .debug else .info,
    };
```

to separe version handling from log-level handing. Can fuse the two, but separately reads better.